northbound/rest/src/test/java/org/eclipse/sensinact/northbound/rest/integration/ResourceAccessTest.java

```diff
@@ -251,10 +252,15 @@ void locationSet(boolean wrapParams) throws Exception {
         assertEquals(204, result.statusCode);
         ResponseGetDTO response = utils.convert(result, ResponseGetDTO.class);
 
+        Thread.sleep(500);
```

Comment:
Yes - without it the check on line `260-261` is unstable. The push update on line `246` is complete, but if we connect to the session fast enough we can still catch some of the lifecycle/data events from it.
```suggestion
        // Wait for any lifecycle events from this provider to complete to
        // to avoid them showing up later
        Thread.sleep(500);
```